lib/compliance/parsepdf417-us.js

```diff
@@ -63,6 +72,12 @@ function parseRaw (data) {
   return normalize(result, version, issuingAuthorityNumber)
 }
 
+function getCountry (state) {
+  return _.findKey(s => {
+    return _.includes(state, s)
+  }, states)
+}
+
```

Comment:
```suggestion
const getCountry = (state) => _.findKey(_.includes(state), states)
```

Reply:
Yeah, that would make more sense, thanks!